### PR TITLE
Feature/audited baal

### LINKED ISSUE #880: Update Initialize tx on Summon
## Update Initialize On Summon

With some changes via Clubhouse and DaoHaus partnership, we are adding [GSN](https://opengsn.org/) to our `Baal` Contract. This change adds the `trustedForwarder` to our initilaization params on summon in the contracts, which we will have to be adjusted in our submit transactions for the summon app

This update is still a WIP on clubhouse's and our side, but once it goes in, we will have to update the Baal packages and initialize summon together.

### LINKED ISSUE #982: Deploy Updated Contracts, Subgraphs and Update UI
Make the changes required after the results from the audit

- [ ] baal-contracts
- [x] moloch-v3-summoner-contract.ts - summon update
- [x] abi-utilities - new abis
- [x] contracts.ts
- [x] subgraph: abis
- [x] subgraph: config
- [x] subgraph: locked admins fields
- [x] subgraph: lock events
- [x] subgraph: contract version field (0.4.1 vs. 1.0.0)
- [x] summoner: new init param
- [x] safe-summoner: new init param
- [ ] deploy subgraph

### PR DESCRIPTION
**Pull request by skuhlmann**
## GitHub Issue

fixes #982 
fixes #880 

## Changes

see list here:
https://github.com/HausDAO/daohaus-monorepo/issues/982

## Packages Added

n/a

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: libs/dao-data/src/Query.ts [hunk 1]
```diff
@@ -83,8 +83,17 @@ import { createPaging, DEFAULT_RECORDS_PER_PAGE } from './utils';
 export default class Query {
   public endpoints: KeychainList;
 
-  constructor() {
+  constructor(graphApiKeys?: Keychain) {
     this.endpoints = ENDPOINTS;
+
+    if (graphApiKeys) {
```
Comment (santteegt) [inline libs/dao-data/src/Query.ts:90]: is this still pending?

Comment (skuhlmann) [reply libs/dao-data/src/Query.ts:90]: @santteegt whoops!

Yeah I better implement that lol

File: apps/summon-app/src/layouts/SummonerForm.tsx [hunk 2]
```diff
@@ -62,6 +62,8 @@ export const SummonerForm = ({
     try {
       const args = assembleTxArgs(formValues, chainId);
 
+      console.log('args', args);
```
Comment (santteegt) [inline apps/summon-app/src/layouts/SummonerForm.tsx]: cleanup console.log

File: apps/summon-app/src/utils/summonTx.ts [hunk 3]
```diff
@@ -217,10 +218,14 @@ export const assembleTxArgs = (
   }
 
   const { POSTER } = handleKeychains(chainId);
+
```
Comment (santteegt) [inline apps/summon-app/src/utils/summonTx.ts:221]: cleanup console.log

File: apps/summon-app/src/utils/summonTx.ts [hunk 4]
```diff
@@ -217,10 +218,14 @@ export const assembleTxArgs = (
...
+  console.log('POSTER', POSTER);
```
Comment (santteegt) [inline apps/summon-app/src/utils/summonTx.ts]: cleanup console.log

File: apps/summon-app/src/utils/summonTx.ts [hunk 5]
```diff
@@ -217,10 +218,14 @@ export const assembleTxArgs = (
...
   const initParams = encodeValues(
-    ['string', 'string'],
-    [tokenName, tokenSymbol]
+    ['string', 'string', 'address'],
+    [tokenName, tokenSymbol, ZERO_ADDRESS]
   );
+
+  console.log('initParams', initParams);
```
Comment (santteegt) [inline apps/summon-app/src/utils/summonTx.ts]: cleanup console.log

**Review by santteegt — APPROVED** [review]

**Pull request merged**